app/src/main/java/org/mozilla/focus/ext/Bitmap.kt

```diff
@@ -26,3 +31,18 @@ fun Bitmap.arePixelsAllTheSame(): Boolean {
 
     return true
 }
+
+fun Bitmap.withRoundedCorners(cornerRadius: Float): Bitmap? {
+    // Need copy of screenshot bitmap because it is immutable
+    val newBitmapWithScreenshotDims = Bitmap.createBitmap(width, height, config)
+    val canvas = Canvas(newBitmapWithScreenshotDims)
+    val paint = Paint()
+
+    // Need to set isAntiAlias to true because it smooths out the edges of what is being drawn
+    paint.isAntiAlias = true
+
+    // Need the shader to replicate the edge color if the shader draws outside of its original bounds
```

Comment:
nit: I don't think we use this functionality. I would remove this comment (unless you think it's helpful).